include/z64save.h

```diff
@@ -143,11 +169,11 @@ typedef struct {
     /* 0x13EA */ u16 unk_13EA; // also alpha type?
     /* 0x13EC */ u16 unk_13EC; // alpha type counter?
     /* 0x13EE */ u16 unk_13EE; // previous alpha type?
-    /* 0x13F0 */ s16 unk_13F0; // magic related
-    /* 0x13F2 */ s16 unk_13F2; // magic related
-    /* 0x13F4 */ s16 unk_13F4; // magic related
-    /* 0x13F6 */ s16 unk_13F6; // magic related
-    /* 0x13F8 */ s16 unk_13F8; // magic related
+    /* 0x13F0 */ s16 magicState; // Determines magic meter behavior on each frame
+    /* 0x13F2 */ s16 magicStateStored; // Stores the previous magic state while magic is increasing. Allows magicState to be restored afterwards
```

Comment:
bump on the suggestion for naming this `magicStateAfterFill` (or `nextMagicStateAfterFill` ?) (maybe Fill->Increase ?)

Reply:
I'm still not sure of associating this specifically with fill, since it's used with magic jars when adding instead of filling (which yes calls fill specifically to set this variable before overwritting the state)

Reply:
Yeah I know, I was thinking "fill" as in "anything that fills up magic even just a bit" but that indeed may be confusing with fill being already used as fully-fill

Maybe (next)`MagicStateAfter`(Increase/Add) then?

Reply:
Yeah, "fill" in magic context specifically means "increase to full capacity", while "add" in magic context specifically means "anything that fills up magic even a bit" (I comment because a different word that "fill" can be used here if you have any other suggestions).

Something like `magicStateAfterIncrease` would capture both cases